Formula/alertmanager.rb

```diff
@@ -0,0 +1,88 @@
+class Alertmanager < Formula
+  desc "Prometheus Alertmanager"
+  homepage "https://prometheus.io/docs/alerting/alertmanager/"
+  url "https://github.com/prometheus/alertmanager/archive/v0.20.0.tar.gz"
+  sha256 "4789ef95b09ba86a66a2923c3535d1bfe30a566390770784c52052c7c83ee1bc"
+
+  depends_on "go" => :build
+
+  def install
+    mkdir_p buildpath/"src/github.com/prometheus/alertmanager"
+    ln_sf buildpath, buildpath/"src/github.com/prometheus/alertmanager"
+
+    system "make", "build"
+    bin.install %w[alertmanager amtool]
+  end
+
+  def post_install
+    (var/"alertmanager").mkdir
+
+    (etc/"alertmanager.sh").write <<~EOS
+      "#{opt_bin}/alertmanager" \
+      --config.file="#{etc}/alertmanager.yml" \
+      --web.listen-address="127.0.0.1:9093" \
+      --storage.path="#{var}/alertmanager"
+    EOS
+
+    alertmanager = (etc/"alertmanager.sh")
+    alertmanager.chmod 0755
+
+    (etc/"alertmanager.yml").write <<~EOS
```

Comment:
Same for this. Either upstream ships a default config file, but we won't provide our own. If upstream decides in the future to change defaults, this could leave us open to any bug/issue with an outdated config.